morf-postgresql/src/main/java/org/alfasoftware/morf/jdbc/postgresql/PostgreSQLDialect.java

```diff
@@ -694,7 +695,7 @@ public Collection<String> alterTableChangeColumnStatements(Table table, Column o
 
     boolean alterNullable = oldColumn.isNullable() != newColumn.isNullable();
     boolean alterType = oldColumn.getType() != newColumn.getType() || oldColumn.getScale() != newColumn.getScale() || oldColumn.getWidth() != newColumn.getWidth();
-    boolean alterDefaultValue = oldColumn.getDefaultValue() != newColumn.getDefaultValue();
+    boolean alterDefaultValue = !Objects.equals(oldColumn.getDefaultValue(), newColumn.getDefaultValue());

```

Comment:
Note: `java.util.Objects.equals()` goes further than `!=`, and uses `Object.equals()`.
Therefore this is a potential behaviour change and must be investigated.

`Column.getDefaultValue()` returns a String. Objects.equals sounds like a better choice.